docs/requirements.md

```diff
@@ -2,11 +2,23 @@
 
 Running Codacy on a Kubernetes cluster requires the following:
 
--   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
+-   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#hardware-requirements)
 -   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
 -   A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
 
-## Cluster resource requirements
+## Hardware Requirements
+
+The resource usage for Codacy depends a lot on the amount of concurrent analysis that you configure. Check the following table for some examples.
+
+| Installation type              | Max. parallel analysis | vCPUs | Memory | Storage |
+| ------------------------------ | ---------------------- | ----- | ------ | ------- |
+| Kubernetes Small Installation  | 2                      | 16    | 64 GB  | 1 TB+   |
+| Kubernetes Medium Installation | 4                      | 32    | 128 GB | 1 TB+   |
+| Kubernetes Big Installation    | 10                     | 64    | 256 GB | 1 TB+   |
+| MicroK8s Minimum               | 2                      | 16    | 32 GB  | 1 TB+   |
+| MicroK8s Recommended           | 2                      | 16    | 64 GB  | 1 TB+   |
+
+### Calculate
```

Comment:
@pauloribeiro-codacy I am thinking on developing here a bit on how to calculate resources

Reply:
I think that the best is if we can abstract this but still provide guidance for customers to provision their infrastructure. Your table above is a great step in this direction! We should invest in this kind of presenting the information to customers so that we don't need to go deep into the technical details. :+1:

Reply:
I think that we no longer need more specific instructions on how to calculate resources. The architecture introduction and the examples presented in the table should be enough now.